release_dashboard/lib/main.dart

```diff
@@ -57,3 +59,29 @@ class MyApp extends StatelessWidget {
     );
   }
 }
+
+/// Widget upon clicking cleans the current release in progress.
```

Comment:
```suggestion
Button to clean the current release.
```

The various tenses makes this hard to read